tests/replication/_base.py

```diff
@@ -371,7 +371,7 @@ def make_worker_hs(
             config=worker_hs.config.server.listeners[0],
             resource=resource,
             server_version_string="1",
-            max_request_body_size=4096,
+            max_request_body_size=8192,
```

Comment:
Was this just an issue in tests? What do we have this set to in real operation?

Reply:
This is just an issue with the tests, I think https://github.com/matrix-org/synapse/blob/3841900aaa8666d193bb6506397d331fbf958233/synapse/app/_base.py#L640 generally handles this in real operation.